docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -21,24 +21,37 @@ aliases:
-
-This page provides instructions for configuring a development environment for working with `viam-micro-server`, outlines the steps for performing various development tasks, and provides troubleshooting and development tips to help organize and streamline work.
+[The Micro-RDK](https://github.com/viamrobotics/micro-rdk/) is the lightweight version of Viam's full Robot Development Kit, designed for resource-limited embedded systems (ESP32) that cannot run the fully-featured `viam-server`.
+`viam-micro-server` is the pre-built firmware built from the Micro-RDK and a default set of {{< glossary_tooltip term_id="module" text="modules" >}}, provided for convenience.
+If you want to use different modules with the Micro-RDK, you can build your own firmware using the instructions on this page.
 
 {{< alert title="Looking to install viam-micro-server?" color="note" >}}
 If you only want to install the pre-built `viam-micro-server` firmware with a default set of modules, follow the normal [setup instructions](/operate/get-started/setup/) instead.
 {{< /alert >}}
 
+{{< expand "Why does the Micro-RDK work differently from the full RDK?" >}}
+Microcontrollers do not have full operating systems like single-board computers and general-purpose computers.
+This means that it is generally harder to make software portable across multiple microcontrollers, and it means that microcontrollers can only run one "program" at a time.
+
+Rather than being able to download and install an executable the way you can on a computer or single-board computer, a microcontroller must be flashed with firmware containing the entire logic to run.
+
+The Micro-RDK is a version of the RDK library written in Rust and designed to run on microcontrollers.
+The Micro-RDK includes built-in support for several common hardware components, such as standard DC motors and an ultrasonic sensor.
+Viam provides a default firmware build that includes the Micro-RDK as well as two modules: a wifi sensor and a memory heap sensor.
+You may well want to use additional modules, in which case you will need to build your own custom firmware, that includes the Micro-RDK as well as one or more Micro-RDK-compatible modules of your own creation or that others have shared.
+{{< /expand >}}
+
 The instructions below are for configuring a development environment in order to:
 
-- Develop custom firmware which combines `viam-micro-server` with one or more modules.
-- Develop modules for `viam-micro-server`.
-- Develop `viam-micro-server` itself.
+- Develop custom firmware which combines the Micro-RDK with one or more modules.
+- Develop modules for the Micro-RDK.
+
+For advanced topics including development of the Micro-RDK itself, to add features and fix bugs, or to produce a build for a specific ESP-IDF version or platform for which Viam does not offer a pre-built solution, see [Viam Micro-RDK Development on GitHub](https://github.com/viamrobotics/micro-rdk/blob/main/DEVELOPMENT.md).
```

Comment:
```suggestion
For advanced topics including development of the Micro-RDK itself, see [Viam Micro-RDK Development on GitHub](https://github.com/viamrobotics/micro-rdk/blob/main/DEVELOPMENT.md).
```

Suggesting this because (1) it generally makes sense that for adding features or fixing bugs, a user would go to the github, and (2) it might be confusing to see "pre-built" when referring to ESP-IDF version, since we're mostly using "pre-built" when talking about viam-micro-server. (I'm also not totally sure what the pre-built ESP-IDF part is referring to 😅).